wave_vision/include/wave/vision/matcher/descriptor_matcher.hpp

```diff
@@ -89,7 +89,7 @@ class DescriptorMatcher {
       const cv::Mat &descriptors_2,
       const std::vector<cv::KeyPoint> &keypoints_1,
       const std::vector<cv::KeyPoint> &keypoints_2,
-      cv::InputArray mask = cv::noArray()) const = 0;
+      cv::InputArray mask) const = 0;
```

Comment:
A foolproof way is to make a function with the same name that takes 4 arguments. Have it call the version with 5 arguments.